Report/Chapters/Process.tex

```diff
@@ -20,7 +20,7 @@ \subsection{Process tasks}
 
 \subsection{Implementation and design tasks}
 
-For the very first part of the project, Martijn, Kristel and Derk were assigned to start working on the program's front end design. Wander and Jan-Jelle were to tackle the beginnings of a usable back end, that is, getting the program designed in the front end to run. This split kept existing during most of the project.
+For the very first part of the project, Martijn, Kristel and Derk were assigned to start working on the program's front end design. Wander and Jan-Jelle were to tackle the beginnings of a usable back end, that is, getting the program designed in the front end to run. This split kept existing during most of the project because of the familiarity with the code.
```

Comment:
> This split kept existing during most of the project because of the familiarity with the code.

Not just code, familiarity with the subjects we already worked on, can't really think of the correct word, but its more than just code.